### DISCUSSION TITLE
ADempiere steering meeting June 24, 2022

### DISCUSSION THREAD

**Discussion by rvergara** [adempiere/adempiere#3930] [Steering Meetings]
### **Meeting Info**
**Date:** June 10, 2022
**Time**: 19:00 GMT ~ 20:00 GMT
**Location:** Discord meeting room
**Meeting URL:** None
**Recorder:** No
**Meeting Objetive:** Review of Project and admin activities to be performed

### **Participants:**

- @e-Evolution
- @marcalwestf
- @yamelsenih
- @eduardolopezuy
- @rvergara
- @niclopup 
- Other attendees at Discord meeting room

### **Meeting agenda**

1. Set minimum temporary decision rules for project
2. Decide on immediate and midterm project web presence
3. Discuss other project community tools

### **Points discussed**

1. Minimum decision rules were discussed. It was agreed that temporarily (a month), project decisions will be taken by the group of major contributors (V. Perez, Y. Senih and M. Calderon). All decisions will require to be made unanimously. Decision rule changes and/or expansion of   project decision makers will require approval of the current major contributor group.
2. short and mid term web presence for the project was discussed. It was decided that short term web presence will be based on current Liferay content, V. Lopez and H. Miranda will work during June 25 th and 26th in order to have the web page uploaded and available on the internet. Support for maintaining the short term web presence available and edited with new content will be led by E. Lopez (openUp) with support from R.Vergara (OFB). Mid term web presence will be based on GitHub pages, for this purpose Y.Senih will configure a structure for content on GitHub. A call for all project partcipant will be then effective for upto date new project content
3. An exploration for a tool for project knowledge base will be performed before next meeting so a proposal can be reviewed. Y Senil will review the funcionality of Stack Oveerflow and investigate uploading of SourceForge forum content
4. ADempiere funcionality documentation was discussed. It was decided to be placed on GitHub. Configuration of GitHub for this task will be supported by Y Senih, content provision will be led by A. Abudinen with participation of volunteers from project members
5. Other project tools (e.g project Wiki) will be discussed during next meeting, however usage of GitHub tools will be prioritized
6. It was decided that R. Vergara should continue his function as "Pro Tempore" steering meetings secretary for one month

### **Next meeting**
Next follow up meeting has been scheduled on July 8th, 2022 at 19:00 GMT via Discord

**Comment by yamelsenih** [comment]
The first page was added here: https://adempiere-site.erpya.com/about/ now we are waiting for @eduardolopezuy and @rvergara  for transfer all content from old page to new before next meeting

**Reply by eduardolopezuy** [reply]
Thanks @yamelsenih for your work creating Github Pages,
For clarity, the proposal we made to have a new website in 3-4 days from the meeting, similar to what we have now in Liferay, but in Wordpress, was not approved.
It was decided to build a new page using Github Pages, that we will add content in english, we together with the @rvergara OFB team throughout the month of July, it was never decided before next friday's meeting.
